packages/jsii/lib/project-info.ts

```diff
@@ -138,6 +140,8 @@ export async function loadProjectInfo(projectRoot: string, { fixPeerDependencies
 
         excludeTypescript: (pkg.jsii && pkg.jsii.excludeTypescript) || [],
         projectReferences: pkg.jsii && pkg.jsii.projectReferences,
+        tscOutDir: pkg.jsii && pkg.jsii.tscoutdir,
```

Comment:
Why not call it `tscOutDir` in the jsii config as well?

(thinking aloud) does this make sense in `package.json` or should they be command-line arguments? I guess we're mirroring the `tsconfig.json` here, right?

Reply:
I named it according to the `outdir` variable but I am happy to rename it. I think this is something I want to configure for my project. I don't think there is much difference in configuring it inside the `jsii` block or in the scripts section ...

Reply:
I think I'd like to have options that'll land in `tsconfig.json` modeled in a separate namespace altogether:

```js
/// package.json
{
  // ...
  "jsii": {
    // ...
    "tsc": {
      "outDir": "dist",
      "rootDir": "lib",
    },
    // ...
  },
  // ...
}
```

Then, I'd use the exact same name that is in `tsconfig`'s `compilerOptions` block.

Reply:
Sounds great. I will update this.